.github/workflows/package-apisix-runtime-deb-openresty-1.21.yml

```diff
@@ -0,0 +1,88 @@
+name: package apisix-runtime deb for debianbullseye-slim with openresty 1.21
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Tag name for the release'
+        required: true
+        default: ''
+
+jobs:
+  build:
+    timeout-minutes: 60
+    env:
+      BUILD_APISIX_RUNTIME_VERSION: 1.1.2
+      ARCH: amd64
+    strategy:
+      matrix:
+        platform:
+          - runner: ubuntu-22.04
+            arch: amd64
+          - runner: buildjet-2vcpu-ubuntu-2204-arm
+            arch: arm64
+    runs-on: ${{ matrix.platform.runner }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: openresty/1.21.4
+
+      - name: Set env.ARCH if platform.arch is aarch64
+        run: |
+          if [ "${{ matrix.platform.arch }}" == "arm64" ]; then
+            echo "ARCH=arm64" >> $GITHUB_ENV
+          fi
+
+      - name: Check arch
+        run: |
+          echo "ARCH: ${{ env.ARCH }}"
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install -y make ruby ruby-dev rubygems build-essential
+
+      - name: Build apisix-runtime deb
+        run: |
+          if [ "${{ matrix.platform.arch }}" == "arm64" ]; then
+            make package type=deb app=apisix-runtime runtime_version=${BUILD_APISIX_RUNTIME_VERSION} image_base=debian image_tag=bullseye-slim arch=linux/arm64/v8
```

Comment:
this already include openssl3, right?

Reply:
yes. If you look above, I have checked out openresty/1.21 branch and that branch builds openresty1.21 with openssl 3.0